### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by mihailfox**

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Globalization;
+using System.Collections.Generic;
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Globalization;
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        public static float ConvertStringToFloat(string message)
+        {
+            Console.WriteLine(message);
+
+            return float.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
+
+        }
+        public static float Bmi(float weight, float heigth)
+        {
+            return weight / (float)Math.Pow(heigth, 2);
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Lesson3.cs]: As far as I have understood the homework those functions should be part of your Program.cs.
You definitly need to create a Demo() function here and fill in what is in the Main() function of Program.cs.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -1,14 +1,40 @@
...
+                Console.WriteLine($"{name} {surename} is {years} old, his weight is {weight} kg and his height is {heigth} cm");
+
+                double bmi = Math.Round(Convert.ToDouble(weight) / Math.Pow(Convert.ToDouble(heigth) / 100.0, 2), 2, MidpointRounding.AwayFromZero);
+
+                Console.WriteLine($"Your BMI is: {bmi}\n");
+
+                Console.WriteLine("press ENTER to continue...\n");
+                Console.ReadLine();
+                currentPerson++;
+            }
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: You clearly can write this function nicer now.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 3 above)
Comment (mihailfox) [reply Src/BootCamp.Chapter/Lesson3.cs]: At the the time I finished the homework this was not in the requirements.

File: Src/BootCamp.Chapter/Program.cs (hunk 4 above)
Comment (mihailfox) [reply Src/BootCamp.Chapter/Program.cs]: Clearly, but homework requirements changed after I finished the initial ones. I will fix it today.
Cheers!

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -1,14 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Move all from main to Lesson3 as mentioned in homework assignment.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

File: Src/BootCamp.Chapter/Program.cs (hunk 5 above)
Comment (mihailfox) [reply Src/BootCamp.Chapter/Program.cs]: > Move all from main to Lesson3 as mentioned in homework assignment.

Moved program logic to class Lesson3

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        public static string ConvertStringToString(string message)
+        {
+            Console.Write(message);
+            return Console.ReadLine();
+        }
+
+        public static float ConvertStringToFloat(string message)
+        {
+            Console.Write(message);
+
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Lesson3.cs]: This empty line can be removed.

Comment (mihailfox) [reply Src/BootCamp.Chapter/Lesson3.cs]: > This empty line can be removed.

Removed the empty line + others.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845597&bid=16083575)

**Pull request closed without merging**